### PR TITLE
chore: CODEOWNERS update proposition

### PR DESCRIPTION
**Pull request by doubleface**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1,14 +1,14 @@
-packages/babel-preset-cozy-app @cpatchane @cedricmessiant
-packages/browserslist-config-cozy @cpatchane @Crash--
-packages/commitlint-config-cozy @enguerran @cedricmessiant
-packages/cozy-app-publish @cpatchane @gregorylegarec @y-lohse
-packages/cozy-ci @y-lohse @cedricmessiant
+packages/babel-preset-cozy-app @edas
+packages/browserslist-config-cozy @Crash--
+packages/commitlint-config-cozy @edas @y-lohse
```
Comment (edas) [inline CODEOWNERS]:
```suggestion
packages/commitlint-config-cozy @y-lohse
```

**Comment by y-lohse** [comment]
@ptbrowne @drazik now that you're back, do you ant to grab any of the repos? If not I think we can merge this!

**Comment by drazik** [comment]
Just added myself to some packages. Thanks for the ping @y-lohse

File: CODEOWNERS [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 packages/cozy-device-helper @ptbrowne
-packages/cozy-doctypes @ptbrowne
-packages/cozy-flags @ptbrowne @cedricmessiant
-packages/cozy-harvest-lib @gregorylegarec @cpatchane
-packages/cozy-interapp @gregorylegarec
+packages/cozy-doctypes @ptbrowne @drazik
+packages/cozy-flags @ptbrowne @drazik
+packages/cozy-harvest-lib @Crash-- @y-lohse
+packages/cozy-interapp @Crash-- @y-lohse
 packages/cozy-realtime @edas
```
Comment (ptbrowne) [inline CODEOWNERS]:
```suggestion
packages/cozy-realtime @edas @ptbrowne
```

**Comment by ptbrowne** [comment]
OK for me :)

**Pull request merged**